python/ray/data/_internal/util.py

```diff
@@ -685,6 +690,7 @@ def unify_block_metadata_schema(
     """
     # Some blocks could be empty, in which case we cannot get their schema.
     # TODO(ekl) validate schema is the same across different blocks.
+    from ray.data._internal.arrow_ops.transform_pyarrow import unify_schemas
```

Comment:
What's the motivation for moving this import?

Reply:
Circular deps